attributes/default.rb

```diff
@@ -58,6 +58,18 @@
 default['cfncluster']['intelmpi']['version'] = '2019.7.217'
 default['cfncluster']['intelmpi']['modulefile'] = "/opt/intel/impi/#{node['cfncluster']['intelmpi']['version']}/intel64/modulefiles/mpi"
 
+# Arm Performance Library
+default['cfncluster']['armpl']['version'] = '20.2.1'
+default['cfncluster']['armpl']['gcc']['version'] = '9.3'
+default['cfncluster']['armpl']['gcc']['url'] = "http://ftp.gnu.org/gnu/gcc/gcc-#{node['cfncluster']['armpl']['gcc']['version']}.0/gcc-#{node['cfncluster']['armpl']['gcc']['version']}.0.tar.gz"
```

Comment:
It looks like the two version numbers defined above are only used in installation. If we ever wanted to update either of these versions we'd have to change tests.rb and all of the modulefiles. Instead can these variables be used in tests.rb, and can the modulefiles be turned into templates? That way upgrading a version is as easy as changing a single line (at least in the ideal case).

Reply:
Done

Reply:
As we discussed offline, I keep the url until S3 distribution.